modules/Library/library_manage_catalog_addProcess.php

```diff
@@ -57,30 +55,26 @@
     } else {
         $replacement == 'N';
     }
-    $comment = $_POST['comment'];
-    $gibbonSpaceID = null;
-    if ($_POST['gibbonSpaceID'] != '') {
-        $gibbonSpaceID = $_POST['gibbonSpaceID'];
-    }
-    $locationDetail = $_POST['locationDetail'];
-    $ownershipType = $_POST['ownershipType'];
+    $comment = $_POST['comment'] ?? '';
+    $gibbonSpaceID = $_POST['gibbonSpaceID'] ?? null;
+
+    $locationDetail = $_POST['locationDetail'] ?? '';
+    $ownershipType = $_POST['ownershipType'] ?? '';
     $gibbonPersonIDOwnership = null;
     if ($ownershipType == 'School' and $_POST['gibbonPersonIDOwnershipSchool'] != '') {
-        $gibbonPersonIDOwnership = $_POST['gibbonPersonIDOwnershipSchool'];
+        $gibbonPersonIDOwnership = $_POST['gibbonPersonIDOwnershipSchool'] ?? '';
     } elseif ($ownershipType == 'Individual' and $_POST['gibbonPersonIDOwnershipIndividual'] != '') {
-        $gibbonPersonIDOwnership = $_POST['gibbonPersonIDOwnershipIndividual'];
+        $gibbonPersonIDOwnership = $_POST['gibbonPersonIDOwnershipIndividual'] ?? '';
     }
-    $gibbonDepartmentID = null;
-    if ($_POST['gibbonDepartmentID'] != '') {
-        $gibbonDepartmentID = $_POST['gibbonDepartmentID'];
-    }
-    $bookable = $_POST['bookable'];
-    $borrowable = $_POST['borrowable'];
-    $status = $_POST['status'];
-    $physicalCondition = $_POST['physicalCondition'];
+    $gibbonDepartmentID = $_POST['gibbonDepartmentID'] ?? null;
```

Comment:
@SKuipers would probably like it better if the fallback is '' instead of `null`.